deployment/setup_20.04.sh

```diff
@@ -91,9 +92,9 @@ RAILS_ENV=production bundle exec rake db:reset
 rake assets:precompile RAILS_ENV=production
 EOF
 
-cp /home/chatwoot/chatwoot/deployment/chatwoot-web.1.service /etc/systemd/system/chatwoot-web.1.service
-cp /home/chatwoot/chatwoot/deployment/chatwoot-worker.1.service /etc/systemd/system/chatwoot-worker.1.service
-cp /home/chatwoot/chatwoot/deployment/chatwoot.target /etc/systemd/system/chatwoot.target
+mv /home/chatwoot/chatwoot/deployment/chatwoot-web.1.service /etc/systemd/system/chatwoot-web.1.service
```

Comment:
I would prefer to  use `cp` here so as to have the original files remain at the location. Our update instructions have steps to copy these files again in case these files are modified to be on the safe side.

https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm#upgrading-to-a-newer-version-of-chatwoot

Any reason why you suggest having `mv` here ?